java/ql/test/utils/modelgenerator/dataflow/p/Sinks.java

```diff
@@ -11,6 +11,12 @@
 
 public class Sinks {
 
+  public Object tainted;
+
+  // Defined as a sink in the model file next to the test.
+  // neutral=p;Sinks;sink;(Object);summary;df-generated
```

Comment:
Why is it neutral when it has a manual model?

Reply:
Same reason as above.